components/notes/note.tsx

```diff
@@ -14,19 +13,21 @@ export interface INote {
 
 const Note = ({ title, slug, date, basePath }: Omit<INote, 'description'>) => {
   return (
-    <Link href={`/${basePath}/[id]`} as={`/${basePath}/${slug}`} passHref>
-      <StyledNoteLink>
-        <StyledNote>
-          <time>
-            <span className="date-icon">
-              <Calendar className="icon" />
-            </span>
-            <p className="date-written">{date.toString()}</p>
-          </time>
-          <h2>{title}</h2>
-        </StyledNote>
-      </StyledNoteLink>
-    </Link>
+    <StyledNoteLink
+      href={`/${basePath}/[id]`}
+      as={`/${basePath}/${slug}`}
+      passHref
+    >
+      <StyledNote>
+        <time>
+          <span className="date-icon">
+            <Calendar className="icon" />
+          </span>
+          <p className="date-written">{date.toString()}</p>
+        </time>
+        <h2>{title}</h2>
+      </StyledNote>
+    </StyledNoteLink>
```

Comment:
maybe we also do not need this change after your changes in [`eslint-config-upleveled@4.1.1`](https://github.com/upleveled/eslint-config-upleveled/releases/tag/v4.1.1)?